Source/Core/Common/MemoryUtil.cpp

```diff
@@ -88,20 +88,21 @@ void FreeMemoryPages(void* ptr, size_t size)
 {
   if (ptr)
   {
-    bool error_occurred = false;
-
 #ifdef _WIN32
     if (!VirtualFree(ptr, 0, MEM_RELEASE))
-      error_occurred = true;
+    {
+      PanicAlert("FreeMemoryPages failed!\n"
+                 "VirtualFree: %s",
+                 GetLastErrorMsg().c_str());
```

Comment:
All PanicAlerts can be one-lined if I use a single string. Is that what you want?